main/zoe/api/Zoe-Intro.md

```diff
@@ -19,32 +19,41 @@ This is an introduction to Zoe, Agoric's smart contract framework. The Zoe Guide
+1. Parties use those invites to make offers.
+1. Determining when an offer *matches* another offer.
+1. Either succeeding or failing to make the trade.
 
 
-An offer’s completion only affects that offer and the assets involved in it; essentially its contract instance. The contract it was made on keeps running, waiting for and processing new offers. 
+An offer’s completion only affects that offer and the assets involved in it. The contract instance it was made on keeps running, waiting for and processing new offers.
 
 
-*Offer safety* is automatically enforced for parties in a contract; if an offer succeeds and completes, all parties get what they wanted. If it fails to complete successfully for any reason, parties keep what they offered.
+*Offer safety* is automatically enforced for parties in a contract; when an offer *completes*, it can complete giving the requesting party what they wanted, or refunding what it offered.
 
 
-All parties in a Zoe contract receive a *seat*, which lets them take action in that contract. They also receive a *promise* for a future payout. The contract itself does not have direct access to any parties’ assets. The payments for an offer are all run through Zoe and its *escrow* capabilities.  
+All parties in a Zoe contract instance receive an invite, which lets them make one offer to zoe for that contract instance. As a result of the offer, they receive a *promise* for a future payout. The contract itself does not have direct access to any parties’ assets. The payments for an offer are all run through Zoe, which *escrows* these assets.
 
 
-Zoe has two distinct APIs. One is client/user-facing, with `zoe.` prefixes on its commands. The other is only exposed to contracts built on Zoe (its *contract facet*), with `zcf.` prefixes on its commands. . 
+Zoe has two distinct APIs. One is client/user-facing zoe *service* object. We'll hold a remote promise for the zoe servive object in a variable named `zoeP` in our code examples.  The other is only exposed to contracts built on Zoe (its *contract facet*), which we'll hold in a variable named `zcf` in our code examples.
 
 
 ## Starting a contract on Zoe
 
 
-While there can be more than one Zoe instance installed and running at any time, for now we’ll focus on the single instance running on Agoric’s testnet. Of course, before you can run a smart contract on Zoe, you have to install it there. For Zoe purposes, *installation* means installing the code your contract is written in (JavaScript and of course the ERTP and Zoe API commands we’ve defined) on a Zoe instance. 
+While there can be more than one Zoe instance installed and running at any time, for now we’ll focus on the single instance running on Agoric’s testnet. Of course, before you can run a smart contract on Zoe, you have to install it there. For Zoe purposes, *installation* means installing the code your contract is written in (JavaScript and of course the ERTP and Zoe API commands we’ve defined) on a Zoe instance.
```

Comment:
I wouldn't say 'you have to install it there'. I could go with 'you have to install it.' or 'you have to tell zoe about the contract' if you want to be less formal.

I would also reword 'installing the code your contract is written in'

```suggestion
While there can be more than one Zoe instance installed and running at any time, for now we’ll focus on the single instance running on Agoric’s testnet. Of course, before you can run a smart contract on Zoe, you have to install the code. For Zoe purposes, *installation* means providing the code for your contract (in JavaScript and of course using the ERTP and Zoe API commands we’ve defined) to a Zoe instance.
```